src/android/CodePush.java

```diff
@@ -518,7 +522,18 @@ private void navigateToFile(File startPageFile) throws MalformedURLException {
     private void navigateToURL(String url) {
         if (url != null) {
             CodePush.ShouldClearHistoryOnLoad = true;
-            this.mainWebView.loadUrlIntoView(url, false);
+            if (this.hasIonicWebViewEngine()) {
+                try {
+                    String ionicWebViewEngineUrlPath = new URI(url).getPath();
+                    String ionicWebViewEngineServerPath = ionicWebViewEngineUrlPath.substring(0, ionicWebViewEngineUrlPath.indexOf("/index.html"));
+                    this.setServerBasePath(ionicWebViewEngineServerPath);
+                    return;
+                } catch (URISyntaxException e) {
+                    e.printStackTrace();
```

Comment:
@atftech-artyom @whegar , Could you please replaced `e.printStackTrace();` with `Utilities.logException(e);` as `Utilities.logException(e);` was implemented for this task?